src/js/hooks/cycleCount/useResolveStep.js

```diff
@@ -340,12 +340,18 @@ const useResolveStep = () => {
     forceRerender();
   };
 
-  const cancelCounts = async (cycleCountRequestIdsToDelete) => {
+  const cancelCounts = async (cycleCountRequestIdsToDelete, cycleCountsIds) => {
```

Comment:
Please rename this `cycleCountsIds` param. Imho it is too close to `cycleCountIds`. I was trying to understand what is happening in this line, and I needed a brief moment to realize that `cycleCountIds` is not the same thing as `cycleCountsIds`:
```
const updatedCycleCountsIds = cycleCountIds
        .filter((id) => !cycleCountsIds.includes(id));
```